README.md

```diff
@@ -1,15 +1,194 @@
-# react-lib-template
-A template that aims to make the implementation of `React` component packages easier and more methodic.
+# react-contentful-rich-text
+
+[![NPM version][npm-image]][npm-url] [![Downloads][downloads-image]][npm-url] [![Build Status][build-status-image]][build-status-url] [![Coverage Status][codecov-image]][codecov-url] [![Dependency status][david-dm-image]][david-dm-url] [![Dev Dependency status][david-dm-dev-image]][david-dm-dev-url]
+
+[npm-url]:https://npmjs.org/package/@moxy/react-contentful-rich-text
+[downloads-image]:https://img.shields.io/npm/dm/@moxy/react-contentful-rich-text.svg
+[npm-image]:https://img.shields.io/npm/v/@moxy/react-contentful-rich-text.svg
+[build-status-url]:https://github.com/moxystudio/react-contentful-rich-text/actions
+[build-status-image]:https://img.shields.io/github/workflow/status/moxystudio/react-contentful-rich-text/Node%20CI/master
+[codecov-url]:https://codecov.io/gh/moxystudio/react-contentful-rich-text
+[codecov-image]:https://img.shields.io/codecov/c/github/moxystudio/react-contentful-rich-text/master.svg
+[david-dm-url]:https://david-dm.org/moxystudio/react-contentful-rich-text
+[david-dm-image]:https://img.shields.io/david/moxystudio/react-contentful-rich-text.svg
+[david-dm-dev-url]:https://david-dm.org/moxystudio/react-contentful-rich-text?type=dev
+[david-dm-dev-image]:https://img.shields.io/david/dev/moxystudio/react-contentful-rich-text.svg
+
+A react component that transforms a contentful rich text structure to HTML.
+
+## Installation
+
+```sh
+$ npm install @moxy/react-contentful-rich-text
+```
+
+This library is written in modern JavaScript and is published in both CommonJS and ES module transpiled variants. If you target older browsers please make sure to transpile accordingly.
+
+## Motivation
+
+Content management systems usually provide a custom text field that lets users define headings, font styles, lists, quotes, links, assets and many more.
+
+Each CMS provides this rich text in a specific format, usually a data structure in json, to be interpreted. Contentful is one of those content management systems.
```

Comment:
Just a small tip, it's good practice when using acronyms for the first time in a text to put it directly after its complete form, inside some parenthesis. In this case, I would just add `(CMS)` to the first paragraph: `Content Management Systems (CMS) usually provide a custom text...`. This way, anyone who reads the rest of the text has the context of what the acronym means.